src/main/java/org/kuali/kfs/kim/document/IdentityManagementPersonDocument.java

```diff
@@ -482,6 +484,17 @@ public void doRouteStatusChange(final DocumentRouteStatusChange statusChangeEven
         if (getDocumentHeader().getWorkflowDocument().isProcessed()) {
             setIfGroupsEditable();
             setIfRolesEditable();
+
+            // ==== CU Customization KFSPTS-27162 Set default for activeFromDate field ===
+            for (final PersonDocumentRole role : roles) {
+                for (final KimDocumentRoleMember rolePrncpl : role.getRolePrncpls()) {
+                    if (ObjectUtils.isNull(rolePrncpl.getActiveFromDate())) {
+                        rolePrncpl.setActiveFromDate(getDateTimeService().getCurrentTimestamp());
```

Comment:
With this solution this will update all the role active from dates to "today" that don't previously have an active from date.  Should we check with the functionals on this?
I think we only want to set the active from date on new records